src/styles.css

```diff
@@ -42,38 +51,88 @@ input {
   padding: 3px 6px;
 }
 
+button,
+button:focus {
+  background-color: #d1f7ff;
+  border-radius: 4px;
+  border: 2px solid #bfb49c;
+  width: 80px;
+  padding: 3px 6px;
+  box-shadow: 4px 4px 4px #7f7868;
+  cursor: pointer;
+}
+
+button:hover {
+  background-color: #d1e0ff;
+  box-shadow: 1px 1px 2px #7f7868;
+}
+
 section {
   display: flex;
   flex-direction: column;
   min-width: 40%;
 }
 
 aside {
-  min-width: 30%;
+  min-width: 40%;
 }
 
 .result {
   display: flex;
   justify-content: space-between;
-  padding: 0.5rem;
+  padding: 4px 8px;
 }
 
 .result:first-of-type {
-  background-color: rgb(155, 155, 155);
+  background-color: #bfb49c;
+  border-top-left-radius: 8px;
+  border-top-right-radius: 8px;
+}
+
+.result:last-of-type {
+  border-bottom-left-radius: 8px;
+  border-bottom-right-radius: 8px;
 }
 
 .results :nth-child(2n+3) {
-  background-color: rgb(155, 155, 155);
+  background-color: #bfb49c;
+}
+
+.results {
+  border: 2px solid #5f5a4e !important;
+  margin-bottom: 2rem;
+  border-radius: 8px;
 }
 
 .your-name,
 .your-score {
   padding: 5px;
   margin-bottom: 1rem;
+  border: 2px solid #5f5a4e !important;
+  border-radius: 8px;
 }
 
 #data-submit {
   display: flex;
   flex-direction: column;
   align-items: flex-end;
 }
+
+footer {
+  width: 100%;
+  position: fixed;
+  padding-bottom: 2rem;
+  bottom: 0;
+  text-align: center;
+  font-size: 0.8rem;
+}
```

Comment:
- [ ] Kindly remove the position: fixed from the footer as the footer goes over the leaderboard scores.
![Screenshot from 2022-05-12 23-37-33](https://user-images.githubusercontent.com/33752735/168172271-375631b4-386f-46df-8208-c7b9fec336f6.png)

Reply:
Hi @ShahierNashaat, Thanks a lot for the kind review! Suggested change has been implemented.